### DISCUSSION TITLE
Where is c1.3.0 ?

### DISCUSSION THREAD

**Discussion by Qui-Gon-JooL** [SynthstromAudible/DelugeFirmware#2715] [Q&A] [answered]
Hi, where can I download  c1.3.0 version ? Please

**Comment by seangoodvibes** [comment]
c.1.3.0 is the unofficial nightly release, which is an experimental build and will have bugs. Some builds may be buggier or less buggier than others. This is the nature of the nightly build as it's being continuously worked on.

it's built on a daily basis if there has been any changes since the last time it was built

It can be download here: https://github.com/SynthstromAudible/DelugeFirmware/releases

But here is the link as well:
https://github.com/SynthstromAudible/DelugeFirmware/releases/download/nightly/nightly.zip

**Comment by Qui-Gon-JooL** [comment]
ok, thank you, because the update of the night indicates 1.2.0, and I cannot find the function indicated in the details of 1.3.0, like the iteration at the same time as the probabilities...
I also can't find the automatic loading of samples during preview...
I dont understand...

**Reply by seangoodvibes** [reply]
> ok, thank you, because the update of the night indicates 1.2.0, and I cannot find the function indicated in the details of 1.3.0, like the iteration at the same time as the probabilities... I also can't find the automatic loading of samples during preview... I dont understand...

have you read the documentation on those features?

https://github.com/SynthstromAudible/DelugeFirmware/blob/community/docs/features/note_noterow_editor.md

https://github.com/SynthstromAudible/DelugeFirmware/blob/community/docs/community_features.md#4312---auto-load-sample-when-browsing

**Comment by Qui-Gon-JooL** [comment]
yes I read the documentation on these features, that's what makes me want to try them, but the behavior is the same as before, nothing more happens: 
Holding a note/row of notes and rotating the selection encoder changes either iterations or probabilities. (like before)
The note editor menu does not open.

I followed the tutu video for the autoloading of samples, nothing new either...

I don't understand what I'm doing wrong, do I have to activate these options somewhere? I don't see it in the features.

I'm using last night's version...2024/09/27

**Reply by seangoodvibes** [reply, marked as answer]
> yes I read the documentation on these features, that's what makes me want to try them, but the behavior is the same as before, nothing more happens: Holding a note/row of notes and rotating the selection encoder changes either iterations or probabilities. (like before) The note editor menu does not open.
> 
> I followed the tutu video for the autoloading of samples, nothing new either...
> 
> I don't understand what I'm doing wrong, do I have to activate these options somewhere? I don't see it in the features.
> 
> I'm using last night's version...2024/09/27

Do you have two bin files on your SD card? If you tried to flash the firmware with two bin files on your SD card it's probably likely that it is not installing the new one and just re-installing the old one.

**Comment by Qui-Gon-JooL** [comment]
well yes, well seen, I hadn't deleted an old .bin, I feel stupid, everything works, thank you.
